### PR TITLE
e2e - test fixes

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/pageObjects/browser-page.ts [hunk 1]
```diff
@@ -219,18 +220,19 @@ export class BrowserPage {
...
         await t.click(this.addKeyNameInput);
         await t.typeText(this.addKeyNameInput, keyName);
         await t.click(this.jsonKeyValueInput);
-        await t.typeText(this.jsonKeyValueInput, value);
+        await t.typeText(this.jsonKeyValueInput, value, { paste: true });
         if (TTL !== undefined) {
             await t.click(this.keyTTLInput);
             await t.typeText(this.keyTTLInput, TTL);
         }
+        await t.debug();
```
Comment (elenaNaboko) [inline tests/e2e/pageObjects/browser-page.ts:235]: Can you please remove debug here?

File: tests/e2e/tests/regression/browser/add-keys.e2e.ts [hunk 2]
```diff
@@ -24,7 +24,8 @@ fixture `Different JSON types creation`
         await cliPage.sendCommandInCli(commandString);
         await deleteDatabase(ossStandaloneConfig.databaseName);
     });
-test('Verify that user can create different types(string, number, null, array, boolean) of JSON', async t => {
+// skipped due the issue https://redislabs.atlassian.net/browse/RI-2966
+test.skip('Verify that user can create different types(string, number, null, array, boolean) of JSON', async t => {
```
Comment (elenaNaboko) [inline tests/e2e/tests/regression/browser/add-keys.e2e.ts:28]: Issue was already fixed, could you please validate?

File: tests/e2e/pageObjects/browser-page.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/pageObjects/browser-page.ts]: done

File: tests/e2e/tests/regression/browser/add-keys.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/browser/add-keys.e2e.ts]: done

**Pull request merged**